.github/labels.yml

```diff
@@ -0,0 +1,219 @@
+- name: comment me
+  color: "8F1B19"
+  description: Request for comments
+
+- name: dependencies
+  color: "0366d6"
+  description: Pull requests that update a dependencies
+
+- name: do not merge yet
+  color: "1EB2F5"
+  description: PR is not ready to be merged yet
+
+- name: do not review yet
+  color: "D44389"
+  description: Do not review yet
+
+- name: documentation
+  color: "0366d6"
+  description: Pull requests that update a documentation
+
+- name: draft
+  color: "E29E35"
+  description: Draft
+
+- name: duplicate
+  color: "cfd3d7"
+  description: This issue or pull request already exists
+
+- name: enhancement
+  color: "a2eeef"
+  description: New feature or request
+
+- name: good first issue
+  color: "7057ff"
+  description: Good for newcomers
+
+- name: help wanted
+  color: "008672"
```

Comment:
Do we really need a label for this? `help wanted` from who? I think the `fyi` labels could work instead of this